runners_manager/runner/Runner.py

```diff
@@ -80,21 +88,29 @@ def toJson(self):
 
         return d
 
-    def update_status(self, elem):
-        if elem['status'] == 'online' and elem['busy'] is True:
-            status = 'running'
-        else:
-            status = elem['status']
-
+    def update_status(self, status: str):
         if self.status == status:
             return
 
-        if self.status == 'offline' and status != 'offline':
+        if self.status not in ['online', 'running'] and status in ['online', 'running']:
             self.started_at = datetime.datetime.now()
```

Comment:
This will create problems
Calling `update_status` 2 time with 'oneline'  or 'running' will overwrite the first `datetime.now()`

This problem was already here before your edit :sweat_smile:

Reply:
Forget it

Reply:
ok lol